learningresources/models.py

```diff
@@ -136,12 +136,25 @@ class LearningResource(BaseModel):
     xa_histogram_grade = models.FloatField(default=0)
     url_name = models.TextField(null=True)
 
-    def get_preview_url(self):
-        """Create a preview URL."""
+    def get_preview_url(self, org=None, course_number=None, run=None):
```

Comment:
I wonder if this would be cleaner as a standalone function since we're now taking in all the arguments we need explicitly.